src/main/java/frc/robot/commands/AutoScoreAndBalance.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commands;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Calibrations;
+import frc.robot.subsystems.Lift;
+import frc.robot.subsystems.drive.DriveSubsystem;
+
+public class AutoScoreAndBalance extends SequentialCommandGroup {
+    private PathPlannerTrajectory traj = PathPlanner.loadPath("Trajectory",
+            new PathConstraints(Calibrations.PLACEHOLDER_DOUBLE, Calibrations.PLACEHOLDER_DOUBLE));
+    public AutoScoreAndBalance(boolean isFirstPath, Lift lift, DriveSubsystem drive) {
+        addCommands(
+                new InstantCommand(lift::scoreGamePiece, lift),
```

Comment:
Hm, it occurs to me this command will instantly finish, meaning the robot will start driving right away. We might want to do a RunCommand instead, and supply a function to check when the lift is done scoring.

`new RunCommand(lift::scoreGamePiece, lift).until(lift::doneScoring)`

That `doneScoring` would need to return a boolean